src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Round.java

```diff
@@ -12,6 +13,9 @@
 
   private int roundNumber;
 
+  // ID(s) of the player(s) that played the highest card(s)
+  @Transient Collection<Long> TrickWinnerIds = new ArrayList<>();
```

Comment:
in java würde ich das feld mit kleinem buchstaben beginnend benennen. (camelCasing).
+ wieso nicht gleich Collection<Participation>, dann kommt eine IRI heraus.